aeron-client/src/main/c/util/aeron_platform.h

```diff
@@ -26,9 +26,8 @@
 
     #if defined(_M_X64)
         #define AERON_CPU_X64 1
-
     #else
-        #error Unsupported CPU!
+        #define AERON_CPU_X86 1
```

Comment:
This is not a supported build option for us and should be removed.

Reply:
resoved.